CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Version 1.28.1/0.49b1 (2024-11-08)
 
+- Fix crash exporting a log record with None body
```

Comment:
@lzchen do I need to fix this to be under a new "Unreleased" heading?

Reply:
Yup, I think usually release branches are made from main which have "unreleased" section already but since you are doing a patch (and the patch release workflow isnt exactly working) youll have to add this manually.

Reply:
Done